pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -253,6 +253,9 @@ public void testTextSearch()
         "SELECT INT_COL, QUERY_LOG_TEXT_COL FROM MyTable WHERE TEXT_MATCH(QUERY_LOG_TEXT_COL, '\"SELECT dimensionCol2\"') LIMIT 50000";
     testTextSearchSelectQueryHelper(query, 11787, false, null);
 
+    query = "SELECT INT_COL, QUERY_LOG_TEXT_COL FROM MyTable WHERE QUERY_LOG_TEXT_COL LIKE '\"SELECT dimensionCol2\"' LIMIT 50000";
```

Comment:
So this syntax is same as Lucene query syntax to keep TextMatchFilterOperator happy. Does SQL LIKE also have a concept of phrase queries using double quotes ?

Reply:
You are right, LIKE does not have phrase queries, nor does it support any of Lucene's syntaxes. Fixed now, thanks